src/Symfony/Component/DependencyInjection/Compiler/AutowirePass.php

```diff
@@ -228,7 +228,7 @@ private function createAutowiredDefinition(\ReflectionClass $typeHint, $id)
      * @param string     $id
      * @param Definition $definition
      *
-     * @return \ReflectionClass|null
+     * @return \ReflectionClass|false
```

Comment:
why false ? null is a better return value for the absence of value than using a boolean